src/plugins/analysis/input_vectors/code/input_vectors.py

```diff
@@ -36,15 +36,23 @@ def __init__(self, plugin_administrator, config=None, recursive=True):
         logging.info('Up and running.')
 
     def process_object(self, file_object: FileObject):
-        with TemporaryDirectory(prefix=self.NAME, dir=get_temp_dir_path(self.config)) as tmp_dir:
+        with TemporaryDirectory(prefix=self.NAME, dir=self.config['data_storage']['docker-mount-base-dir']) as tmp_dir:
             file_path = Path(tmp_dir) / file_object.file_name
             file_path.write_bytes(file_object.binary)
             try:
                 result = run_docker_container(
-                    DOCKER_IMAGE, TIMEOUT_IN_SECONDS, CONTAINER_TARGET_PATH, reraise=True,
-                    mount=(CONTAINER_TARGET_PATH, str(file_path)), label=self.NAME, include_stderr=False
+                    DOCKER_IMAGE,
+                    # We explicitly don't want stderr to ignore "Cannot analyse at [...]"
+                    combine_stderr_stdout=False,
+                    logging_label=self.NAME,
+                    timeout=TIMEOUT_IN_SECONDS,
+                    command=CONTAINER_TARGET_PATH,
+                    mounts=[
+                        Mount(CONTAINER_TARGET_PATH, str(file_path), type='bind'),
+                    ],
+                    stderr=False,
```

Comment:
Does `stderr=False` make a difference if `combine_stderr_stdout=False`? (also `False` is the default value for `combine_stderr_stdout`)

Reply:
Good catch, `stderr=False` does not make any difference.
It was a leftover from before I introduced `combine_stderr_stdout`